app/src/main/java/com/example/hangouts/BayesianRating.java

```diff
@@ -0,0 +1,37 @@
+package com.example.hangouts;
+
+import org.apache.commons.math3.distribution.NormalDistribution;
+
+import java.util.List;
+
+public class BayesianRating {
+
+    final static int standardNormalMean = 0;
+    final static int standardStdDeviation = 1;
+    final static NormalDistribution standardNormalDist = new NormalDistribution(standardNormalMean,
+            standardStdDeviation);
+
+    private static double getPpf(double x) {
+        return standardNormalDist.inverseCumulativeProbability(x);
+    }
+
+    public static double getBayesianRating(List<Double> list, double significance) {
+        double sum = 0;
+        int size = list.size();
+        for (int i = 0; i < size; i++) {
+            sum += list.get(i);
+        }
+        if (sum == 0) {
+            return 0;
+        }
+        double z_value = getPpf(1 - (1 - significance) / 2);
```

Comment:
lets avoid underscore in variable names - its not inline with java naming convention.  always camelcase.
zValue